autosar/behavior.py

```diff
@@ -691,9 +691,11 @@ def createRunnable(self, name, portAccess=None, symbol=None, concurrent=False, e
         * exclusiveAreas: List of strings containing which exclusive areas this runnable will access.
           Note: For mode ports you will at best get read access. If you want to set new modes use modeSwitchPoints.
         * modeSwitchPoint: List of strings containing port names that this runnable will explicitly use for setting modes.
+        * minStartInterval: Specifies the time in seconds by which two consecutive starts of an ExecutableEntity are guaranteed to be separated.

```

Comment:
I know the ARXML actually uses seconds for times. However when I designed the class autosar.behavior.TimingEvent I switched to using milliseconds in the Python API as this was more convenient for the users.
With the introduction of the minStartInterval in runnables there is now an inconsistency in the API. One part is using seconds and another part is using milliseconds.
Does it make sense to change minStartInterval to also use milliseconds for consistency?

Reply:
I did not look so much at that, my problem where that I needed to remove the parameter in some cases as it is not supported for all SWC types.

But changing to follow autosar.behavior.TimingEvent sounds like a good solution.